### PR TITLE
Added Trapping Rainwater problem in python

### PR DESCRIPTION
**Pull request by Shalvi-Singhal**
#594 fixed.
@tarun26091999 @plazzy99 please review it.

### CODE REVIEW

**Comment by Rohan27s** [comment]
@Shalvi-Singhal Add print statements that will ask for the size and values of the array from the users

**Comment by Shalvi-Singhal** [comment]
> @Shalvi-Singhal Add print statements that will ask for the size and values of the array from the users

I have added the print statements @Rohan27s please review it.

File: Code/Python/trapping_rainwater.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Problem Description:
```
Comment (Rohan27s) [inline Code/Python/trapping_rainwater.py:1]: use ''' and the comments into multiline comments instead of using # again and again

**Review by Rohan27s — CHANGES_REQUESTED** [review]
Code if fine, just remove unnecessary space and add multiline comment

**Comment by Shalvi-Singhal** [comment]
Changes made.
@Rohan27s

**Review by Rohan27s — APPROVED** [review]
Nice Work, thanks for your contribution!

**Comment by Shalvi-Singhal** [comment]
Thank you!

**Pull request merged**